### PR TITLE
doc: fix new repo links

### PR DESCRIPTION
**Pull request by jgribonvald**
##### Checklist

-   [X] the [individual contributor license agreement][] is signed
-   [X] commit message follows [commit guidelines][]
-   [ ] tests are included
-   [X] documentation is changed or added
-   [ ] [message properties][] have been updated with new phrases
-   [ ] view conforms with [WCAG 2.0 AA][]

##### Description of change

Move old doc links to new Repository links.

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

**Comment by jgribonvald** [comment]
@ChristianMurphy I'm not sure about coberall and travis links, please could you provide me a review on this part ?
Also some rights should be configured, like on gitter for the new repo link and I don't have rights to grant access on this part it seems.

**Review by ChristianMurphy — APPROVED** [review]
Thanks @jgribonvald! LGTM :+1: 

Not sure what the best path forward is for Travis, we can update the link, but the builds may not work, we're close the API limit, if we haven't already hit it https://jamescooke.info/travis-hitting-githubs-api-limits-for-open-source-projects.html
Long term, if we want to patch 4.X releases, we may need to migrate them to GitHub actions or another CI system.

For Coveralls, that is strange :thinking: 
As far as I know it should be reporting stats back.
I wouldn't block this PR for coveralls, that is an issue unto itself.

File: uPortal-security/uPortal-security-core/src/main/java/org/apereo/portal/security/firewall/README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-The `RequestParameterPolicyEnforcementFilter` is a blunt generic Java Servlet Filter suitable for 
+The `RequestParameterPolicyEnforcementFilter` is a blunt generic Java Servlet Filter suitable for
 
  * in general, blocking malicious requests involving unexpectedly duplicate request parameters or request parameters with unexpected and poorly handled characters in their values.
- * in specific, blocking the malicious requests involved in `CVE-2014-4172`, to which certain Java CAS Client versions 
+ * in specific, blocking the malicious requests involved in `CVE-2014-4172`, to which certain Java CAS Client versions
 are vulnerable, including Java CAS Client versions that have been included in uPortal releases.
 
-It is a fork of that offered to the `cas-server-security-filter` project at https://github.com/Jasig/cas-server-security-filter/pull/6 .
+It is a fork of that offered to the `cas-server-security-filter` project at https://github.com/uPortal-Project/cas-server-security-filter/pull/6 .
```
Comment (jonathanmtran) [inline uPortal-security/uPortal-security-core/src/main/java/org/apereo/portal/security/firewall/README.md:10]: This repo is still in the Jasig Github org.

File: uPortal-security/uPortal-security-core/src/main/java/org/apereo/portal/security/firewall/RequestParameterPolicyEnforcementFilter.java [hunk 2]
```diff
@@ -30,7 +30,7 @@
  * do not meet configured rules.
  *
  * <p>This is a fork of the Filter offered at
- * https://github.com/Jasig/cas-server-security-filter/pull/6 .
+ * https://github.com/uPortal-Project/cas-server-security-filter/pull/6 .
```
Comment (jonathanmtran) [inline uPortal-security/uPortal-security-core/src/main/java/org/apereo/portal/security/firewall/RequestParameterPolicyEnforcementFilter.java:33]: This repo is still in the Jasig Github org.

**Review by jonathanmtran — CHANGES_REQUESTED** [review]

File: uPortal-security/uPortal-security-core/src/main/java/org/apereo/portal/security/firewall/README.md (hunk 1 above)
Comment (jgribonvald) [reply uPortal-security/uPortal-security-core/src/main/java/org/apereo/portal/security/firewall/README.md]: Thanks Jonathan, I fixed it with apereo link (removing the redirect from Jasig)

File: uPortal-security/uPortal-security-core/src/main/java/org/apereo/portal/security/firewall/RequestParameterPolicyEnforcementFilter.java (hunk 2 above)
Comment (jgribonvald) [reply uPortal-security/uPortal-security-core/src/main/java/org/apereo/portal/security/firewall/RequestParameterPolicyEnforcementFilter.java]: Thanks Jonathan, I fixed it with apereo link (removing the redirect from Jasig)

**Comment by jgribonvald** [comment]
@bjagg do you know if there is again a need about the `RequestParameterPolicyEnforcementFilter` ? As CAS client evolved I think that the CVE are fixed now, more the `cas-server-security-filter` is deprecated... Or this should be applied with the embedded CAS ? 

So maybe we can create an issue around that to remove this "useless" code now ?

**Comment by jgribonvald** [comment]
> Not sure what the best path forward is for Travis, we can update the link, but the builds may not work, we're close the API limit, if we haven't already hit it https://jamescooke.info/travis-hitting-githubs-api-limits-for-open-source-projects.html
> Long term, if we want to patch 4.X releases, we may need to migrate them to GitHub actions or another CI system.

GitHub actions would be good I think, but not sure how it will evolve as some paid options appear. What is the best ? It's difficult to find a way.

**Comment by ChristianMurphy** [comment]
I'd say for the purposes of updating links, I'd keep your change as is.

> but not sure how it will evolve as some paid options appear

It's true, we're using a free-for-open-source solution and in doing so we rely on the generosity of the hosting platform.
Travis offered a generous plan for a time, GitHub offers a generous plan now, it's possible another platform could be used in the future.

> What is the best ? It's difficult to find a way.

I'm not sure there is a "best" :sweat_smile: 
Most CI/CD are a small wrapper around a terminal shell, and are about the same all things considered.

**Review by jonathanmtran — APPROVED** [review]

**Pull request merged**